rllib/agents/trainer.py

```diff
@@ -763,6 +763,7 @@ def get_scope():
                 # `in_evaluation` flag.
                 evaluation_config.update({
                     "batch_mode": "complete_episodes",
+                    "rollout_fragment_length": 1,
```

Comment:
is this basically the "timesteps_per_iteration" param for training rollouts directly under the config dict?

Reply:
Great question:

"batch_mode": "complete_episodes" makes sure we have at least(!) one episode in the produced batch. However, it could be more than 1 episode, if e.g. rollout_fragment_length=100 and an episode only lasts for 20 timesteps (in this case we would have 5 episodes in the batch!).

To avoid the above (having more episodes in a batch than one would expect via the `complete_episode` setting), we set rollout_fragment_length to 1 so that the episode termination becomes the limiting factor here.